core/src/lib/tls_openssl_private.cc

```diff
@@ -556,3 +609,9 @@ void TlsOpenSsl::SetCipherList(const std::string& cipherlist)
   Dmsg1(100, "Set cipherlist:\t<%s>\n", cipherlist.c_str());
   d_->cipherlist_ = cipherlist;
 }
+
+void TlsOpenSsl::SetProtocol(std::string protocol)
+{
+  Dmsg1(100, "Set protocol:\t<%s>\n", protocol.c_str());
```

Comment:
do we really need that debugging message here? The value isn't "set" here, it is just pushed into the data structure.